### PR TITLE
improve: use public Analytics SDK dependency

### PR DESCRIPTION
**Pull request by donnie-jp**
# Description
Add a dependency on the public hosted version of the Analytics SDK

# Checklist
- [x] I have read the [contributing guidelines](CONTRIBUTING.md).
- [x] I have completed the [SDK Development Learning Path](CONTRIBUTING.md#SDK-Development-Learning-Path) (if submitting a feature PR)
- [ ] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `fastlane ci` without errors

### CODE REVIEW

**Comment by donnie-jp** [comment]
@Climbatize To configure your emulator builds properly you'll need to set the RAT env vars as secrets in your CI.

**Review by corycaywood — APPROVED** [review]
LGTM 👍

File: Podfile.lock [hunk 1]
```diff
@@ -5,60 +5,56 @@ PODS:
...
-    - nanopb (~> 2.30908.0)
-  - GoogleAppMeasurement/AdIdSupport (7.11.0):
+  - GoogleAppMeasurement (7.10.0):
     - GoogleUtilities/AppDelegateSwizzler (~> 7.0)
     - GoogleUtilities/MethodSwizzler (~> 7.0)
     - GoogleUtilities/Network (~> 7.0)
     - "GoogleUtilities/NSData+zlib (~> 7.0)"
     - nanopb (~> 2.30908.0)
   - GoogleUserMessagingPlatform (1.4.0)
-  - GoogleUtilities/AppDelegateSwizzler (7.4.1):
```
Comment (rleojoseph) [inline Podfile.lock:22]: Guess you need to update the pods.

**Review by rleojoseph — APPROVED** [review]
Looks good to me :+1:

**Review by laubengaier — APPROVED** [review]
LGTM 👍

File: Podfile.lock (hunk 1 above)
Comment (donnie-jp) [reply Podfile.lock:22]: @rleojoseph I ran `updatePods` manually and it's also run as part of the `build_simulator` lane that I modified. I just ran it again on my branch and there's no changes to Podfile.lock. Is it ok to go ahead and merge?

Comment (rleojoseph) [reply Podfile.lock:22]: Yeah, I think it's ok.

**Pull request merged**